### PR TITLE
Connect api

### PR DESCRIPTION
**Pull request by DelhinRharl**
# In this pull request,

- [ ]  I added API functions to the project

### CODE REVIEW

File: src/redux/Books.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+export const addRemote = (payload) => async (dispatch) => {
+  const books = await fetch(url, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(payload),
+  });
+  console.log(books);
```
Comment (AshakaE) [inline src/redux/Books.js]: - [ ] Please always remember to delete the `console.log` statements. see [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

File: src/redux/Books.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    };
+  });
+  dispatch(fetchBook(displayBooks));
+};
+
+export const removeRemote = (item_id) => async (dispatch) => {
+  const books = await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/1ACApRNEuGr2czHzTboo/books/${item_id}`, {
+    method: 'DELETE',
+  });
+  console.log(books, item_id);
```
Comment (AshakaE) [inline src/redux/Books.js]: - [ ] Please always remember to delete the `console.log` statements.

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @DelhinRharl  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
## STATUS: APPROVED :taco: :tada: :1st_place_medal: 
Hi :wave:  @DelhinRharl  , 

appr

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**